src/ngraph/op/broadcast.hpp

```diff
@@ -24,92 +24,70 @@ namespace ngraph
             {
             public:
-                static constexpr NodeTypeInfo type_info{"Broadcast", 0};
+                static constexpr NodeTypeInfo type_info{"Broadcast", 3};
                 const NodeTypeInfo& get_type_info() const override { return type_info; }
                 /// \brief Constructs a broadcast operation.
                 Broadcast() = default;
                 /// \brief Constructs a broadcast operation.
                 ///
                 /// \param arg            The input tensor to be broadcast.
-                /// \param shape          The shape of the output tensor.
-                /// \param broadcast_axes The axis positions (0-based) in the result that are being
-                ///                       broadcast. The remaining axes in shape must be the same as
-                ///                       the shape of arg.
+                /// \param target_shape   The shape of the output tensor.
+                /// \param axes_mapping   The axis positions (0-based) in the result that correspond
+                ///                       to input axes. 'Arg' tensor is broadcast along the
+                ///                       remaining
+                ///                       axes.
+                ///                       E.g., Input Shape - [3, 4], Target Shape - [3, 5, 4, 4]
+                ///                       axes_mapping - [0, 2] => Broadcast along axes 1 and 3.
+                ///                       axes_mapping - [0, 3] => Broadcast along axes 1 and 2.
+                /// \param broadcast_spec Broadcast specification to use for determining broadcast
+                ///                       axes. 'axes_mapping' should not be provided if mode other
+                ///                       than explicit (none) is used.
                 Broadcast(const Output<Node>& arg,
-                          const Shape& shape,
-                          const AxisSet& broadcast_axes);
+                          const Output<Node>& target_shape,
+                          const Output<Node>& axes_mapping,
+                          const AutoBroadcastSpec& broadcast_spec = AutoBroadcastSpec());
+
+                /// \brief Constructs a broadcast operation.
+                ///
+                /// \param arg            The input tensor to be broadcast.
+                /// \param target_shape   The shape of the output tensor.
+                /// \param broadcast_spec Broadcast specification to use for determining broadcast
+                ///                       axes
+                Broadcast(const Output<Node>& arg,
+                          const Output<Node>& target_shape,
+                          const AutoBroadcastSpec& broadcast_spec =
+                              AutoBroadcastSpec(AutoBroadcastType::NUMPY));
                 bool visit_attributes(AttributeVisitor& visitor) override;
+                size_t get_version() const override { return 1; }
```

Comment:
What is the reason to override the Node class implementation for this method? If use Node's implementation, this call will return 3. But here the code returns 1, is this intensional?

Reply:
right, thanks it was copy-paste artifact ;)